QUARKUS-3461.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-3461 Verify behaviour and functionality of Quarkus CLI update command with RHBQ
+- JIRA link: https://issues.redhat.com/browse/QUARKUS-3461
+- Quarkus Guides:
+    - https://quarkus.io/guides/update-quarkus
+
+## Scope of the testing
+- Exploratory testing of the feature over Quarkus upstream and last released version of RHBQ
+- Verify that the `quarkus update` command explicitly warns users about deprecated/unsupported extensions during migration to the newer version.
+  In most cases deprecated extensions are moved to `io.quarkiverse`, which is accepted for Quarkus upstream but not for RHBQ.
```

Comment:
How are you going to automate that? I don't think there is a point in testing past releases unless the used recipe is reused and if it is, then you should test the new change, not the old one because there always can be a typo.